app/controllers/login_controller.rb

```diff
@@ -4,13 +4,22 @@
 
 # Handle authentication: login/register
 class LoginController < ApplicationController
-  attr_accessor :logged_in_user
-
-  def initialize(logged_in_user)
-    @logged_in_user = logged_in_user
+  
+  def initialize(request)
+    super(request)
   end
 
-  def login_user(request)
-    User.where(username: request.params['username']).first
+  def attempt_login_user()
```

Comment:
You actually don't need the parenthesis here when not passing any arguments.

Reply:
Oh, but I do ;)